Aptoide/aptoide-client-v8
# [feature/ASV-1549] APPC BN flow

**What does this PR do?**

   This PR adds APPC upgrade functionality to the Apps section. It adds a new section at the top of the fragment and also adds a "See more" fragment if the list of upgrades exceeds 2. Functionally it is similar to an update, but it is technically a downgrade instead (the app must be removed first).

**Database changed?**

   No

**How should this be manually tested?**

  Check if the appc upgrades are working fine (i.e. it uninstalls the current version and installs it again). Check if the progress works ok and if it removes successfully from list after upgrading. Also check this for the See More fragment. Finally, make sure that there are no repeated upgrades/updates. If an app has is marked for APPC upgrade there should never be a normal update for it.

**What are the relevant tickets?**

  [ASV-1549](https://aptoide.atlassian.net/browse/ASV-1549)

**Code Review Checklist**
